src/lib/crypto/kimchi_backend/common/plonk_dlog_proof.ml

```diff
@@ -325,7 +360,18 @@ module Make (Inputs : Inputs_intf) = struct
         ; lookup =
             Option.map t.commitments.lookup
               ~f:(fun l : _ Pickles_types.Plonk_types.Messages.Lookup.t ->
-                { sorted = Array.map ~f:wo l.sorted
+                { sorted =
+                    Vector.init
+                      Pickles_types.Plonk_types.Lookup_sorted_minus_1.n
+                      ~f:(fun i -> (* TODO: Fixme *) wo l.sorted.(i))
```

Comment:
Be more explicit on the FIXME

Reply:
I know that something's not right, but not what. I've just deleted them to unblock the PR.